bats/gql/user-mark-in-app-notification-as-read.gql

```diff
@@ -0,0 +1,10 @@
+mutation userMarkInAppNotificationAsRead(
+  $input: UserMarkInAppNotificationAsReadInput!
+) {
+  userMarkInAppNotificationAsRead(input: $input) {
```

Comment:
I think conventional naming would be `userInAppNotificationMarkAsRead` (ie. verb at the end)

Reply:
renamed